src/Components/DisplayOneArticle/DisplayArticleComments/index.jsx

```diff
@@ -22,34 +30,69 @@ export function DisplayArticleComments({ article_id }) {
       });
   }, [article_id]);
 
+  function addNewComment(newComment) {
+    setArticleComments([newComment, ...articleComments]);
+    setCurrentArticle((currentArticle) => {
+      return {
+        ...currentArticle,
+        comment_count: +currentArticle.comment_count + 1,
+      };
+    });
+    postArticleComment(article_id, {
+      username: newComment.author,
+      body: newComment.body,
+    })
+      .then(() => {
+        setIsSuccessMsgVisible(true);
+        setIsErrorAddCommentMsgVisible(false);
+        return;
+      })
+      .catch(() => {
+        setIsSuccessMsgVisible(false);
+        setIsErrorAddCommentMsgVisible(true);
+      });
+
+    setTimeout(() => {
+      setIsSuccessMsgVisible(false);
+    }, 3000);
+  }
+
   if (isLoading) return <p>Loading...</p>;
 
   if (isError) return <p>Whops, some error here... please reload the page!</p>;
 
   return (
     <div>
-      {articleComments.map((comment) => {
-        return (
-          <li key={comment.comment_id}>
-            <p>Author: {comment.author}</p>
-            <p>
-              Created at:{" "}
-              {comment.created_at
-                ? new Date(comment.created_at)
-                    .toJSON()
-                    .slice(0, 10)
-                    .split("-")
-                    .reverse()
-                    .join("/")
-                : ""}
-            </p>
-            <p>{comment.body}</p>
-            {comment.votes >= 0 && <p>👍{comment.votes}</p>}
-            {comment.votes < 0 && <p>👎{comment.votes}</p>}
-            <hr />
-          </li>
-        );
-      })}
+      <CommentForm
+        addNewComment={addNewComment}
+        isSuccessMsgVisible={isSuccessMsgVisible}
+        isErrorAddCommentMsgVisible={isErrorAddCommentMsgVisible}
+      />
+      <p>Total comments: {total_comments}</p>
+      <ul className="display--article--comments--list">
+        {articleComments.map((comment) => {
+          return (
+            <li key={comment.comment_id}>
+              <p>Author: {comment.author}</p>
+              <p>
+                Created at:{" "}
+                {comment.created_at
+                  ? new Date(comment.created_at)
```

Comment:
There may be a better way to do this. there is a package colled moments I believe that might come to your rescue here